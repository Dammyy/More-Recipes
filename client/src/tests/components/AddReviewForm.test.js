import React from 'react';
import { shallow } from 'enzyme';
import
{ AddReviewForm } from '../../components/AddReviewForm';

describe('AddReviewForm  component', () => {
  describe('Snapshot', () => {
    test('AddReviewForm component should fully render', () => {
      const component = shallow(<AddReviewForm />);
      expect(component).toMatchSnapshot();
    });
  });
});
