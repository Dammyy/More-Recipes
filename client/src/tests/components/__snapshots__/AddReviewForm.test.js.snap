// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddReviewForm  component Snapshot AddReviewForm component should fully render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AddReviewForm />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="panel-title text-center r-d-titles"
        >
          Leave a review
        </h2>,
        <div
          className="panel-body"
        >
          <form
            action=""
            name="form"
            noValidate={true}
            onSubmit={undefined}
          >
            <div
              className="form-group"
            >
              <Field
                className="form-control"
                component="textarea"
                id="review-input-field"
                name="review"
                placeholder="Leave a Review"
                type="text"
              />
            </div>
            <button
              className="btn btn-primary btn-block"
              id="submit-review"
              type="submit"
            >
              Submit
            </button>
          </form>
        </div>,
      ],
      "className": "col-md-12",
      "id": "recipe-display",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Leave a review",
          "className": "panel-title text-center r-d-titles",
        },
        "ref": null,
        "rendered": "Leave a review",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <form
            action=""
            name="form"
            noValidate={true}
            onSubmit={undefined}
          >
            <div
              className="form-group"
            >
              <Field
                className="form-control"
                component="textarea"
                id="review-input-field"
                name="review"
                placeholder="Leave a Review"
                type="text"
              />
            </div>
            <button
              className="btn btn-primary btn-block"
              id="submit-review"
              type="submit"
            >
              Submit
            </button>
          </form>,
          "className": "panel-body",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "action": "",
            "children": Array [
              <div
                className="form-group"
              >
                <Field
                  className="form-control"
                  component="textarea"
                  id="review-input-field"
                  name="review"
                  placeholder="Leave a Review"
                  type="text"
                />
              </div>,
              <button
                className="btn btn-primary btn-block"
                id="submit-review"
                type="submit"
              >
                Submit
              </button>,
            ],
            "name": "form",
            "noValidate": true,
            "onSubmit": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Field
                  className="form-control"
                  component="textarea"
                  id="review-input-field"
                  name="review"
                  placeholder="Leave a Review"
                  type="text"
                />,
                "className": "form-group",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "className": "form-control",
                  "component": "textarea",
                  "id": "review-input-field",
                  "name": "review",
                  "placeholder": "Leave a Review",
                  "type": "text",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Submit",
                "className": "btn btn-primary btn-block",
                "id": "submit-review",
                "type": "submit",
              },
              "ref": null,
              "rendered": "Submit",
              "type": "button",
            },
          ],
          "type": "form",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="panel-title text-center r-d-titles"
          >
            Leave a review
          </h2>,
          <div
            className="panel-body"
          >
            <form
              action=""
              name="form"
              noValidate={true}
              onSubmit={undefined}
            >
              <div
                className="form-group"
              >
                <Field
                  className="form-control"
                  component="textarea"
                  id="review-input-field"
                  name="review"
                  placeholder="Leave a Review"
                  type="text"
                />
              </div>
              <button
                className="btn btn-primary btn-block"
                id="submit-review"
                type="submit"
              >
                Submit
              </button>
            </form>
          </div>,
        ],
        "className": "col-md-12",
        "id": "recipe-display",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Leave a review",
            "className": "panel-title text-center r-d-titles",
          },
          "ref": null,
          "rendered": "Leave a review",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <form
              action=""
              name="form"
              noValidate={true}
              onSubmit={undefined}
            >
              <div
                className="form-group"
              >
                <Field
                  className="form-control"
                  component="textarea"
                  id="review-input-field"
                  name="review"
                  placeholder="Leave a Review"
                  type="text"
                />
              </div>
              <button
                className="btn btn-primary btn-block"
                id="submit-review"
                type="submit"
              >
                Submit
              </button>
            </form>,
            "className": "panel-body",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "action": "",
              "children": Array [
                <div
                  className="form-group"
                >
                  <Field
                    className="form-control"
                    component="textarea"
                    id="review-input-field"
                    name="review"
                    placeholder="Leave a Review"
                    type="text"
                  />
                </div>,
                <button
                  className="btn btn-primary btn-block"
                  id="submit-review"
                  type="submit"
                >
                  Submit
                </button>,
              ],
              "name": "form",
              "noValidate": true,
              "onSubmit": undefined,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Field
                    className="form-control"
                    component="textarea"
                    id="review-input-field"
                    name="review"
                    placeholder="Leave a Review"
                    type="text"
                  />,
                  "className": "form-group",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "className": "form-control",
                    "component": "textarea",
                    "id": "review-input-field",
                    "name": "review",
                    "placeholder": "Leave a Review",
                    "type": "text",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Submit",
                  "className": "btn btn-primary btn-block",
                  "id": "submit-review",
                  "type": "submit",
                },
                "ref": null,
                "rendered": "Submit",
                "type": "button",
              },
            ],
            "type": "form",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
